src/apps/explorer/components/TransanctionBatchGraph/index.tsx

```diff
@@ -103,7 +103,7 @@ export function TransactionBatchGraph(params: GraphBatchTxParams): JSX.Element {
   }
 
   return (
-    <>
+    <div id="tx-graph" style={{ flex: 1 }}>
```

Comment:
why hardcoded?

Reply:
curious, why this change helps?

Reply:
Taking the style updates one layer out solves the re-render. I've made sure to avoid relying on styled components on this one to avoid potential re-renders.